etc/system-lock/lock.sh

```diff
@@ -64,7 +64,7 @@ fi
 sudo chmod +x /etc/rc.local
 fi
 
-if [ $is_success = true ]; then
-sudo reboot
-fi
+#if [ $is_success = true ]; then
+#sudo reboot
```

Comment:
I guess, if this is required for production, it should be in the code.
I think, you can mock (fake) the reboot command by using an alias. What also works is creating your own reboot command and prepending the folder of it to `$PATH`